sdk/spring/scripts/compatibility_insert_dependencymanagement.py

```diff
@@ -90,28 +90,16 @@ def get_prop_position(pom_file_content):
 
 def get_prop_content(pom_file_content, spring_boot_dependencies_version, spring_cloud_dependencies_version):
     if contains_properties(pom_file_content):
-        return get_properties_contend(spring_boot_dependencies_version, spring_cloud_dependencies_version)
+        return get_properties_content(spring_boot_dependencies_version, spring_cloud_dependencies_version)
     else:
-        return get_properties_contend_with_tag(spring_boot_dependencies_version, spring_cloud_dependencies_version)
+        return get_properties_content_with_tag(spring_boot_dependencies_version, spring_cloud_dependencies_version)
 
 
 def add_dependency_management_for_file(file_path, spring_boot_dependencies_version, spring_cloud_dependencies_version):
     spring_cloud_version = spring_cloud_dependencies_version
+    if file_path.replace("/", "").replace("\\", "") == ".sdkspringpom.xml":
+        return
```

Comment:
As I mentioned last time, either we can define a list of ignored_files, or perform the action when there's a `<dependencies>` tag in current pom file.